mp/src/game/shared/neo/weapons/weapon_srs.cpp

```diff
@@ -56,7 +56,7 @@ void CWeaponSRS::ItemPreFrame()
 		m_bRoundChambered = true;
 	}
 
-	if (m_flNextPrimaryAttack <= gpGlobals->curtime && !m_bRoundChambered && !m_bRoundBeingChambered && m_iClip1 > 0)
+	if (m_flLastAttackTime + 0.08f <= gpGlobals->curtime && !m_bRoundChambered && !m_bRoundBeingChambered && m_iClip1 > 0)
```

Comment:
Where is the value 0.08 coming from here?

Reply:
0.08 is the time after firing when the character begins playing the round chamber animation, as I recorded it at 1/10 speed